src/frontend/Sider/Personoversikt/FrittståendeBrev/FrittståendeBrev.tsx

```diff
@@ -45,6 +52,16 @@ const FrittståendeBrev: React.FC<{ valgtStønadstype: Stønadstype; fagsakId: s
         }
     }, [mellomlagretBrev, settBrevmal]);
 
+    useEffect(() => {
+        hentBrevmaler('FRITTSTAENDE');
+    }, [hentBrevmaler]);
+
+    useEffect(() => {
+        if (brevmal) {
+            hentMalStruktur();
```

Comment:
Vet ikke helt hva`hentMalStruktur` gjør. Men burde man kunne flytte den til then i 
`const hentBrevmaler = useCallback(() => {`
Sånn at man henter `hentMalStruktur` på nytt hver gang man henter mal?

Reply:
`hentBrevmaler` henter en liste med tilgjengelige brevmaler, `hentMalStruktur` henter strukturen for den valgte malen. Man vet derfor ikke hvilken mal man skal hente for når hentMaler er ferdig